### PR TITLE
chore: removes squoosh and upgrade packages

### PR DESCRIPTION
**Pull request by marlonmarcello**
## Goal 
Basic housekeeping

1. Upgraded and updated packages
2. Removed squoosh as it's [abandoned](https://www.npmjs.com/package/@squoosh/lib#project-no-longer-maintained) and it's holding us back from updating Node. Replaced it with sharp, it's the fastest and seems to be the most reliable to work with on the new Macs.
3. Updated Node version to latest LTS

## Testing

1. Clone the repo
2. checkout `chore/upgrade`
3. cd into `cli`
4. Link package with `npm link`
5. Create a new directory and cd into it `cd ~/Desktop; mkdir corgi-test; cd corgi-test`
6. Link package `npm link @wethegit/corgi`
7. Bootstrap new project `npx corgi project`
8. Use correct node version with you node version manager
9. Install packages `npm i`
10. Add some images to the `src/images/defined` directory
11. Run dev `npm run dev`

Fell free to try out a couple of `<Image />` components with different parameters as well.

### CODE REVIEW

File: LICENSE.txt
Comment (liamegan) [inline LICENSE.txt:1]: Who is Scott Chacon?

**Review by liamegan — COMMENTED** [review]
Who is Scott Chacon?

File: LICENSE.txt
Comment (marlonmarcello) [reply LICENSE.txt:1]: RIP Scott. Fixed.

**Review by liamegan — APPROVED** [review]
LGTM

**Pull request merged**